### PR TITLE
96 remove hardcoded sensor tag map

### LINKED ISSUE #96: Replace hardcoded sensorTagMap with dynamic storage
## Description
Now that we have implemented #89 we have a storage that maps device mac addresses to their tags. Implemented in [XSensDotMetadataStorage.kt](https://github.com/Sensors-in-Paradise/android-native-xsens/blob/89-show-labels-of-disconnected-xsens-dot-devices/app/src/main/java/sensors_in_paradise/sonar/XSensDotMetadataStorage.kt). 
With this and our new sensor sets we can remove the hardcoded map of tags to mac addressess

## Must Include

- [ ] Replace sensorTagMap in GlobalValues.kt and use [XSensDotMetadataStorage.kt](https://github.com/Sensors-in-Paradise/android-native-xsens/blob/89-show-labels-of-disconnected-xsens-dot-devices/app/src/main/java/sensors_in_paradise/sonar/XSensDotMetadataStorage.kt) instead

### PR DESCRIPTION
**Pull request by tfiedlerdev**
Closes #96 

The sensor set is now inferred by the sensor tag. 
Sensors with an incorrect tag, will be handled as sensors that have no tag

sensors are sorted by their tag in the connection screen

### CODE REVIEW

File: app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt [hunk 1]
```diff
@@ -181,6 +194,7 @@ class LoggingManager(
         resolveMissingFields {
             moveTempFiles(personTV.text.toString(), recordingEndTime)
             labelTV.text = ""
+            personTV.text = ""
```
Comment (kpostnov) [inline app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt:197]: This line was removed in order to not reset the person after every recording.

Comment (kpostnov) [inline app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt:197]: This line was removed in order to not reset the person after every recording.

**Review by kpostnov — CHANGES_REQUESTED** [review]

**Comment by kpostnov** [comment]
The rest looks fine to me.

File: app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt (hunk 1 above)
Comment (tfiedlerdev) [reply app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt]: Well spotted! Fixed now

**Review by kpostnov — APPROVED** [review]

**Pull request merged**